providers/implementations/include/prov/names.h

```diff
@@ -337,6 +337,22 @@
 #define PROV_NAMES_DSA "DSA:dsaEncryption:1.2.840.10040.4.1"
 #define PROV_DESCS_DSA "OpenSSL DSA implementation"
 #define PROV_NAMES_RSA "RSA:rsaEncryption:1.2.840.113549.1.1.1"
+#define PROV_NAMES_RSA_MD2 "RSA-MD2:md2WithRSAEncryption:1.2.840.113549.1.1.2"
+#define PROV_NAMES_RSA_MD4 "RSA-MD4:md4WithEncryption:1.2.840.113549.1.1.3"
+#define PROV_NAMES_RSA_MD5 "RSA-MD5:md5WithRSAEncryption:1.2.840.113549.1.1.4"
```

Comment:
These should be removed

Reply:
No.  They will serve when/if we do add them in the legacy provider.  In the mean time, they don't hurt anyone